internal/render/render.go

```diff
@@ -18,16 +19,41 @@ var (
 )
 
 func init() {
+	initEmptyTmpls()
+	for _, name := range bindata.AssetNames() {
+		tmpls = template.Must(tmpls.Parse(string(bindata.MustAsset(name))))
+	}
+}
+
+// LoadCustomTemplate allows to load in custom tamplates from a specified path
+func LoadCustomTemplate(path string) error {
+	path = strings.Trim(path, "/")
+	initEmptyTmpls()
+
+	files, err := ioutil.ReadDir(path)
+	if err != nil {
+		return err
+	}
+
+	for _, f := range files {
+		data, err := ioutil.ReadFile(fmt.Sprintf("%s/%s", path, f.Name()))
+		if err != nil {
+			return err
+		}
+		tmpls = template.Must(tmpls.Parse(string(data)))
```

Comment:
Prefer to return an error here instead of using `Must`, since the latter panics. `Must` should only be used in a package's `init` function.